presto-docs/src/main/sphinx/release/release-0.285.rst

```diff
@@ -0,0 +1,76 @@
+
+Documentation Changes
+_____________________
+* Add Presto to MySQL type mapping in :doc:`MySQL Connector</connector/mysql>`.
+* Improve :doc:`REST API</rest>` by providing links.
+* Improve :doc:`Resource Groups </admin/resource-groups>`.
+
+
+General Changes
+_______________
+* Add :func:`array_least_frequent` function.
+* Add support for HANA connector. :pr:`21034`
+* Add task killer which is triggered when a worker is running out of memory and the garbage collector cannot reclaim sufficient memory. Two strategies are provided: full garbage collection, and frequent full garbage collection. :pr:`21254`
+* Add a session property ``history_input_table_statistics_matching_threshold`` to set the threshold for input statistics match in HBO. :pr:`21063`
+* Add a session property ``remove_redundant_cast_to_varchar_in_join`` (enabled by default) to remove redundant `cast to varchar` expressions in a join condition.
+* Add a session property ``enable_hbo_for_scaled_writer`` (disabled by default) to use HBO for scaled writers.
+* Add a session property ``use_partial_aggregation_history`` if partial aggregation statistics must be used to split aggregates into partial and final. :pr:`21160`
+* Add a config property ``task.memory-based-slowdown-threshold`` that provides a heap memory threshold below which new splits are not processed.
+* Rename config ``experiemental.task.high-memory-task-killer-strategy`` to ``experimental.task.high-memory-task-killer-strategy``.
+* Add optimization where values node followed by an always false filter is converted to an empty values node.
+* Add information about cost-based optimizers and the source of stats they use (CBO/HBO) in explain plans when session property ``verbose_optimizer_info_enabled=true``.
+* Add a boolean to the PlanOptimizer's return value to indicate if an optimization has been triggered.
+* Improve latency of HBO. :pr:`21297`
+* Improve HBO to track partial aggregation statistics. :pr:`21160`
+* Improve PushPartialAggregationThroughExchange to use partial aggregation statistics when available. This extends to multi-key aggregations.
+* Improve union node by removing the empty inputs.
+* Skip HBO retry when it timeouts.
+* Improve join-reordering to work with non-simple equi-join predicates.
+* Fix `min_by(x, y, n) / max_by(x, y, n)` functions to ensure `n` is unique.
+* Track null probe keys in join and use it to enable outer join null salt in history based optimization.
+* Update Joda-Time to 2.12.5 to use 2023c tzdata. The JVM should also be updated to ensure the timezone data is consistent. See :pr:`21329` for more details.
+* Update Alluxio to 304 from 2.8.1.
+* Update AWS SDK to 1.12.560 from 1.12.261.
+* Update Avro version to 1.11.3 from 1.9.2.
+
+Hive Connector Changes
+______________________
+* Improve support for schema evolution of partition column types for Parquet file format. :pr:`19983`
+* Add support for Parquet writer versions V1 and V2 in Hive and Iceberg catalogs.
```

Comment:
```suggestion
* Add support for Parquet writer versions V1 and V2 in Hive and Iceberg catalogs. :pr:`20957`
```
I'd link to the doc but it doesn't seem to exist yet per [this comment on PR 21500](https://github.com/prestodb/presto/pull/21500#discussion_r1431890364). Maybe it would be appropriate to create a new issue for adding this requested doc.

Reply:
It was merged here https://github.com/prestodb/presto/pull/21587

Reply:
Thanks for finding that and adding the link to the doc!